tripal_chado/includes/loaders/tripal_chado.pub_importer_AGL.inc

```diff
@@ -892,7 +932,12 @@ function tripal_pub_AGL_parse_pubxml($pub_xml) {
   }
 
   // build the full authors list
-  if (is_array($pub['Author List'])) {
+  if (!array_key_exists('Author List', $pub)) {
+    // there is a constraint in chado.pubprop against value being null 
+    $pub['Author List'] = array(['Surname' => 'anonymous']);
```

Comment:
This is a minor comment but the new Drupal style guideline is for `[]` instead of `array()`, see the [Drupal style guideline](https://www.drupal.org/docs/develop/standards/coding-standards#array).

```suggestion
    $pub['Author List'] = [['Surname' => 'anonymous']];
```